### ISSUE TITLE
Chef 13.2.20 support?

### ISSUE THREAD

**Issue by manuelmazzuola** [sous-chefs/dhcp#84]
### Cookbook version
5.4.0

### Chef-client version
13.2.20

### Platform Details
GCE ubuntu 14.04

```
       ================================================================================          
       Recipe Compile Error in /tmp/kitchen/cache/cookbooks/dhcp/attributes/default.rb           
       ================================================================================          
                                                                                                 
       NoMethodError                                                                             
       -------------                                                                             
       undefined method `domain' for #<Chef::Node::Attribute:0x00000004bdbd80>                   
                                                                                                 
       Cookbook Trace:                                                                           
       ---------------                                                                           
         /tmp/kitchen/cache/cookbooks/dhcp/attributes/default.rb:39:in `from_file'               
                                                                                                 
       Relevant File Content:                                                                    
       ----------------------                                                                    
       /tmp/kitchen/cache/cookbooks/dhcp/attributes/default.rb:                                  
                                                                                                 
        32:  default['dhcp']['group_data'] = {}                                                  
        33:  default['dhcp']['network_data'] = {}                                                
        34:  default['dhcp']['shared_network_data'] = {}                                         
        35:                                                                                      
        36:  defined?(domain) || default['domain'] = 'local'                                     
        37:                                                                                      
        38:  default['dhcp']['parameters']['default-lease-time'] = '6400'                        
        39>> default['dhcp']['parameters']['ddns-domainname'] = "\"#{domain}\""    
```

**Comment by bcg62** [comment]
also seeing this same issue

**Comment by damacus** [comment]
See #89

**Comment by damacus** [comment]
Closed via #89
